be/src/exprs/vectorized/utility_functions.cpp

```diff
@@ -50,4 +50,15 @@ ColumnPtr UtilityFunctions::last_query_id(FunctionContext* context, const Column
     }
 }
 
+ColumnPtr UtilityFunctions::uuid(FunctionContext*, const Columns& columns) {
+    int32_t num_rows = ColumnHelper::get_const_value<TYPE_INT>(columns[columns.size() - 1]);
```

Comment:
function uuid has no arguments.
So the columns should be zero, and this will cause BE crash